doc/interpolation.rst

```diff
@@ -0,0 +1,183 @@
+.. _interp:
+
+Interpolating data
+==================
+
+.. ipython:: python
+   :suppress:
+
+    import numpy as np
+    import pandas as pd
+    import xarray as xr
+    np.random.seed(123456)
+
+xarray offers flexible interpolating routines, which has a similar interface
```

Comment:
flexible interpolati**on** routines, with a similar interface to :ref:`indexing <indexing>`.